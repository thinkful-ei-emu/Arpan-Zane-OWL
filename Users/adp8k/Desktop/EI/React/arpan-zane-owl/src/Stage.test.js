import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage'
import renderer from 'react-test-renderer';


describe('<Stage/>', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<Stage />, div);


        ReactDOM.unmountComponentAtNode(div);

    });

    it('renders this UI as expected', () => {
        // Render the component, as JSON
        const tree = renderer.create(<Stage />).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});