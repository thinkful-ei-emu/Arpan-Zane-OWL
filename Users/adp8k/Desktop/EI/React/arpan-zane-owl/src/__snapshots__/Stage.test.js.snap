// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage/> renders this UI as expected 1`] = `
<div>
  <div>
    <h2>
      Koren Templeton
    </h2>
    <img
      alt="avatar"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <p>
      
    </p>
    <p>
      On Stage
    </p>
  </div>
  <div>
    <h2>
      Ashla Attwool
    </h2>
    <img
      alt="avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <p>
      
    </p>
    <p>
      On Stage
    </p>
  </div>
</div>
`;
